Calculator/CalculatorViewController.swift

```diff
@@ -8,35 +8,53 @@
 
 import UIKit
 
+
 class CalculatorViewController: UIViewController {
     
+    // Creating brain property of typee CalculatorBrain
+    var brain: CalculatorBrain? = nil
```

Comment:
No need for the `= nill` here.
In fact you. have made "brain" optional when you created. No need to assign it a value here. You will be assigning its value later on.